doc/CODING.md

```diff
@@ -51,33 +51,56 @@ intent to add a new rule here.
 
 See [DESIGN](DESIGN.md) document too, they complement each other.
 
+### Coding Style ###
 
-### C Guidelines ###
+- Limits
 
  * Functions should not exceed 100 lines.
  * Files should not exceed 1000 lines.
- * A line should not be longer than 72 characters.
- * Split up when those limits are reached.
- * Rationale: Readability with split windows.
- * The compiler shall not emit any warning (or error).
+ * A line should not be longer than 80 characters.
```

Comment:
> What about 120? Do you really have a workflow/screen/editor where 120 characters are problematic?

I do not know if this questions is directed at me (too), but will answer it anyway. I usually use two windows side by side. With a text width of 80 characters, the editor window takes up a little less than half of my screen. I use the other half for stuff related to the coding process (documentation browser, Markdown preview, web browser, compiler output, second code window,…). A maximum of 120 character breaks this workflow, since I only have a small medium resolution screen (15 inch, 2880x1800).

Here are other reasons why a maximum of 80 characters per line might make sense:
- The default window width of all terminal emulators I know is 80 characters
- All but one (WebKit) of the default coding styles of Clang-Format use this value: LLVM, Google, Chromium, Mozilla
- [IBM punchcards](http://stackoverflow.com/questions/110928/is-there-a-valid-reason-for-enforcing-a-maximum-width-of-80-characters-in-a-code) use a default column width of 80 characters. Why would we use a different value if IBM considered this one ideal in 1928!